click/_termui_impl.py

```diff
@@ -130,7 +130,7 @@ def eta(self):
 
     def format_eta(self):
         if self.eta_known:
-            t = self.eta + 1
+            t = int(self.eta)
```

Comment:
* offset removed as it's not needed when update order has been swapped (update after instead of before).
* The int is here since the code below operates on integer logic, and eta (time) is decimal.
* Note: Touching this line will cause the change to conflict with a PR for issue 485, so depending on merging order, some rebasing will likely be needed.